src/tests/ftest/container/create.py

```diff
@@ -47,51 +45,35 @@ def test_container_create(self):
 
         :avocado: tags=all,container,tiny,smoke,full_regression,containercreate
         """
-        pool = None
         contuuid = None
         expected_results = []
 
-        try:
-            # initialize a python pool object then create the underlying
-            # daos storage
-            createmode = self.params.get("mode", '/run/poolparams/')
-            createuid = os.geteuid()
-            creategid = os.getegid()
-            createsetid = self.params.get("setname", '/run/poolparams/')
-            createsize = self.params.get("size", '/run/poolparams/')
-
-            # setup the pool
-            pool = DaosPool(self.context)
-            pool.create(createmode, createuid, creategid,
-                        createsize, createsetid)
-            pool.connect(1 << 1)
-
-            # maybe use the good handle, maybe not
-            handleparam = self.params.get("handle", '/run/poolhandle/*')
-            if handleparam == 'VALID':
-                poh = pool.handle
-            else:
-                poh = handleparam
-                expected_results.append('FAIL')
-
-            # maybe use a good UUID, maybe not
-            uuidparam = self.params.get("uuid", "/uuids/*")
-            expected_results.append(uuidparam[1])
-            if uuidparam[0] == 'NULLPTR':
-                self.cancel("skipping this test until DAOS-2043 is fixed")
-                # Commenting the line below as it will result in an
-                # AttributeError and never get to the DAOS API code.
-                # Should be further investigated as part of DAOS-3081
-                # contuuid = 'NULLPTR'
-            else:
-                contuuid = uuid.UUID(uuidparam[0])
-
-            should_fail = False
-            for result in expected_results:
-                if result == 'FAIL':
-                    should_fail = True
-                    break
+        # setup the pool
+        self.prepare_pool()
+
+        # maybe use the good handle, maybe not
+        handleparam = self.params.get("handle", '/run/poolhandle/*')
+        if handleparam == 'VALID':
+            poh = self.pool.pool.handle
+        else:
+            poh = handleparam
+            expected_results.append('FAIL')
+
+        # maybe use a good UUID, maybe not
+        uuidparam = self.params.get("uuid", "/uuids/*")
+        expected_results.append(uuidparam[1])
+        if uuidparam[0] == 'NULLPTR':
+            contuuid = 'NULLPTR'
+        else:
+            contuuid = uuid.UUID(uuidparam[0])
```

Comment:
(pylint-redefined-variable-type) Redefinition of contuuid type from str to uuid.UUID